addons/sale_stock/report/sale_order_report_templates.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
     <template id="report_saleorder_document_inherit_sale_stock" inherit_id="sale.report_saleorder_document">
         <xpath expr="//div[@name='expiration_date']" position="after">
-            <div class="col-3" t-if="doc.incoterm" groups="sale_stock.group_display_incoterm">
+            <div class="col-auto col-3 mw-100 mb-2" t-if="doc.incoterm" groups="sale_stock.group_display_incoterm">
                 <strong>Incoterm:</strong>
                 <p t-field="doc.incoterm.code"/>
```

Comment:
```suggestion
                <p class="m-0" t-field="doc.incoterm.code"/>
```
You should also follow the general format for the `<p>` tag then ;).

Reply:
Hello @Feyensv , I have changed my code as per your suggestion.

Thank you :)